pkg/ccl/streamingccl/event.go

```diff
@@ -30,6 +32,9 @@ const (
 	// CheckpointEvent indicates that GetResolvedSpans will be meaningful. The resolved
 	// timestamp indicates that all KVs have been emitted up to this timestamp.
 	CheckpointEvent
+	// SpanConfigEvent indicates that the SpanConfig field of an enven holds an updated
```

Comment:
```suggestion
	// SpanConfigEvent indicates that the SpanConfig field of an event holds an updated
```

Reply:
Done, and also refactored this file a bit.